pkg/polaris/graphql/azure.go

```diff
@@ -58,26 +58,35 @@ type AzureSubscriptionStatus struct {
 // AzureAddCloudAccountWithoutOAuth adds the Azure Subscription cloud account
 // for given feature without OAuth.
 func (c *Client) AzureAddCloudAccountWithoutOAuth(ctx context.Context, cloud AzureCloud, tenantDomain string,
-	regions []AzureRegion, feature CloudAccountFeature, subscriptions []AzureSubscriptionIn, policyVersion int) (string, []AzureSubscriptionStatus, error) {
+	regions []AzureRegion, features []CloudAccountFeature, subscriptions []AzureSubscriptionIn, policyVersion int) (string, []AzureSubscriptionStatus, error) {
 	c.log.Print(log.Trace, "graphql.Client.AzureAddCloudAccountWithoutOAuth")
 
+	var feature CloudAccountFeature
+	if len(features) > 0 {
+		feature = features[0]
+	}
```

Comment:
This and `Feature` and `Features` below is to handle changing `feature` from a value to a list of values.